pkg/minikube/mustload/mustload.go

```diff
@@ -0,0 +1,169 @@
+			out.T(out.Shrug, `There is no local cluster named "{{.cluster}}"`, out.V{"cluster": name})
+			exitTip("start", name, exit.Data)
+		}
+		exit.WithError("Error getting cluster config", err)
+	}
+
+	return api, cc
+}
+
+// Running is a cmd-friendly way to load a running cluster
+func Running(name string) ClusterController {
+	api, cc := Partial(name)
+
+	cp, err := config.PrimaryControlPlane(cc)
+	if err != nil {
+		exit.WithError("Unable to find control plane", err)
+	}
+
+	machineName := driver.MachineName(*cc, cp)
+	hs, err := machine.Status(api, machineName)
+	if err != nil {
+		exit.WithError("Unable to get machine status", err)
+	}
+
+	if hs == state.None.String() {
+		out.T(out.Shrug, `The control plane node "{{.name}}" does not exist.`, out.V{"name": cp.Name})
+		exitTip("start", name, exit.Unavailable)
+	}
+
+	if hs == state.Stopped.String() {
+		out.T(out.Shrug, `The control plane node must be running for this command`)
+		exitTip("start", name, exit.Unavailable)
+	}
+
+	if hs != state.Running.String() {
+		out.T(out.Shrug, `The control plane node is not running (state={{.state}})`, out.V{"name": cp.Name, "state": hs})
+		exitTip("start", name, exit.Unavailable)
+	}
+
+	host, err := machine.LoadHost(api, name)
+	if err != nil {
+		exit.WithError("Unable to load host", err)
+	}
+
+	cr, err := machine.CommandRunner(host)
+	if err != nil {
+		exit.WithError("Unable to get command runner", err)
+	}
+
+	ips, err := host.Driver.GetIP()
+	if err != nil {
+		exit.WithError("Unable to get driver IP", err)
+	}
+
+	if driver.IsKIC(host.DriverName) {
+		ips = oci.DefaultBindIPV4
+	}
+
+	ip := net.ParseIP(ips)
+	if ip == nil {
+		exit.WithCodeT(exit.Software, fmt.Sprintf("Unable to parse driver IP: %q", ips))
+	}
+
+	return ClusterController{
+		API:      api,
+		Config:   cc,
+		CPRunner: cr,
+		CPHost:   host,
+		CPNode:   &cp,
+		DriverIP: ip,
+	}
+}
+
+// Healthy is a cmd-friendly way to load a healthy cluster
+func Healthy(name string) ClusterController {
+	co := Running(name)
+
+	as, err := kverify.APIServerStatus(co.CPRunner, net.ParseIP(co.CPNode.IP), co.CPNode.Port)
+	if err != nil {
+		out.T(out.FailureType, `Unable to get control plane status: {{.error}}`, out.V{"error": err})
+		exitTip("delete", name, exit.Unavailable)
+	}
+
+	if as == state.Paused {
+		out.T(out.Shrug, `The control plane for "{{.name}}" is paused!`, out.V{"name": name})
+		exitTip("unpause", name, exit.Unavailable)
+	}
+
+	if as != state.Running {
+		out.T(out.Shrug, `This control plane is not running! (state={{.state}})`, out.V{"state": as.String()})
+		out.T(out.Warning, `This is unusual - you may want to investigate using "{{.command}} logs"`, out.V{"command": minikubeCmd(name)})
+		exitTip("start", name, exit.Unavailable)
+	}
+	return co
+}
+
+// Return a minikube command containing the current profile name
+func minikubeCmd(cname string) string {
+	if cname != constants.DefaultClusterName {
+		return fmt.Sprintf("minikube -p %s", cname)
```

Comment:
not sure if -p is better before or after start
```
minikube -p p1 start
```
or 
```
minikube start -p p1
```
but we need to be consistant in all minikube, in docker-env we use -p after. 
so either make this like others, or make others like this.